examples/vmod_be/README.md

```diff
@@ -19,18 +19,11 @@ import be;
 import be from "path/to/libbe.so";
 ```
 
-### Object `parrot`
+### Constructor (`parrot`) `parrot be.parrot(STRING to_repeat)`
 
 parrot is our VCL object, which just holds a rust Backend,
 it only needs two functions:
 - new(), so that the VCL can instantiate it
 - backend(), so that we can produce a C pointer for varnish to use
 
-```vcl
-// Create a new instance of the object in your VCL init function
-sub vcl_init {
-    new new = parrot.new(STRING to_repeat);
-}
-```
-
-#### Method `BACKEND backend()`
+#### Method `BACKEND .backend()`
```

Comment:
I think having an example of how to create an object auto-generated for novice users as part of the docs is a good thing to have, wouldn't it?

Reply:
yes and no, I can definitely see people copy-and-pasting blindly and being shocked that the code doesn't compile

Reply:
I am sure we can come up with some clearer syntax like `/* STRING to_repeat, */` or something similar - but this is still much better starting point than doing it from scratch, i.e. figuring out that it has to be part of `vcl_init`

Reply:
I disagree, giving the wrong code is worse that giving no code. Two things we can do though:
- point at https://varnish-cache.org/docs/trunk/reference/vmod.html#objects-and-methods
- let the vmod writer provide examples
we don't have to reinvent the universe to make that pie, docs exist and it's ok to assume some familiarity with vmods

Reply:
I am not saying we should give **working** code. What I am proposing is to give something that looks like code - a path I have seen in many demos/documentations, and clearly state what has to be done to make the code usable, i.e. some commented out parameters with regular text

Reply:
oki, perhaps you are right - lets force the user to actually write proper docs anyway :)